package.json

```diff
@@ -4,9 +4,27 @@
 	"description": "AeroGear Services Web SDK package",
 	"main": "index.js",
 	"scripts": {
-		"test": "lerna run test"
+		"test": "lerna run test",
+		"test-ci": "npm-run-all test coverage:*",
```

Comment:
This one failed locally for me. Just a FYI, no big deal.

Reply:
Great catch!